### PR TITLE
Only show course owners at course homepage

### PR DESCRIPTION
**Pull request by harryggg**
issue #933

### CODE REVIEW

**Comment by lowjoel** [comment]
No, this isn't a fix. You've just changed the definition of "instructors" across the entire application. Think carefully before making changes.

**Comment by harryggg** [comment]
The only time you used 'instructor' is in course show view....

**Comment by harryggg** [comment]
would it be better if I make a new scope called 'managers' and use manager in homepage? (manager contains owner and managers)

**Comment by harryggg** [comment]
@lowjoel changed, sorry for the trouble made

**Comment by lowjoel** [comment]
@allenwq what do you think about this?

File: app/views/course/courses/show.html.slim [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
   h2 = t('.instructors')
   div
-    = render partial: 'layouts/user', collection: current_course.instructors.map(&:user)
+    = render partial: 'layouts/user', collection: current_course.managers.map(&:user)
```
Comment (allenwq) [inline app/views/course/courses/show.html.slim]: eager load user ? `current_course.managers.includes(:user)`

Comment (harryggg) [inline app/views/course/courses/show.html.slim]: okie, thanks for the advice :D

**Comment by allenwq** [comment]
@harryggg Thanks !

@lowjoel Looks good.

**Comment by lowjoel** [comment]
@harryggg specs?

**Comment by harryggg** [comment]
specs are added

**Comment by lowjoel** [comment]
@harryggg there's also specs on each of the course user roles.

**Comment by harryggg** [comment]
@lowjoel I just realise I shouldn't name the new scope 'managers', it will cause confusion with the role 'manager'... but what should I name the new scope...Or should I just use  select method in view directly...

**Comment by allenwq** [comment]
@harryggg Maybe you call the new scope `manager_and_owner` ?  No you should not use select, t's slow and we should filter the users at db level.

**Comment by harryggg** [comment]
@allenwq okie, thank you so much!

**Comment by harryggg** [comment]
@lowjoel spec is added, thx!

**Comment by lowjoel** [comment]
Merged in ebb757ca8e3609d711887aedbd40a66dac143220

**Pull request closed without merging**